src/content/docs/es/core-concepts/astro-syntax.mdx

```diff
@@ -0,0 +1,191 @@
+</div>
+```
+
+### Atributos dinámicos
+
+Las variables locales pueden ser usadas en llaves para pasarle atributos para ambos, elementos HTML y componentes:
+
+```astro title="src/components/DynamicAttributes.astro" "{name}" "${name}"
+---
+const name = "Astro";
+---
+<h1 class={name}>Attribute expressions are supported</h1>
+
+<MyComponent templateLiteralNameAttribute={`MyNameIs${name}`} />
+```
+
+:::caution
+Los atributos HTML serán convertidos a strings, para que no sea posible pasarle funciones y objetos a los elementos HTML. 
+Por ejemplo, no puedes asignar un controlador de eventos a un elemento HTML en un componente de Astro:
+
+```astro
+---
+// dont-do-this.astro
+function handleClick () {
+    console.log("button clicked!");
+}
+---
+<!-- ❌ This doesn't work! ❌ -->
+<button onClick={handleClick}>Nothing will happen when you click me!</button>
+```
+
+En lugar de eso, usa un código del lado del cliente para añadir el controlador de eventos, como harías en JavaScript vanilla.
+
+```astro
+---
+// do-this-instead.astro
+---
+<button id="button">Click Me</button>
+<script>
+  function handleClick () {
+    console.log("button clicked!");
+  }
+  document.getElementById("button").addEventListener("click", handleClick);
+</script>
+```
+:::
+
+### HTML dinámico
+
+Las variables locales pueden ser usadas en funciones similares a JSX para producir elementos HTML generados dinámicamente.
+
+```astro title="src/components/DynamicHtml.astro" "{item}"
+---
+const items = ["Dog", "Cat", "Platypus"];
+---
+<ul>
+  {items.map((item) => (
+    <li>{item}</li>
+  ))}
+</ul>
+```
+
+Astro pueden mostrar HTML condicionalmente usando operadores lógicos y expresiones ternarias de JSX.
+
+```astro title="src/components/ConditionalHtml.astro" "visible"
+---
+const visible = true;
+---
+{visible && <p>Show me!</p>}
+
+{visible ? <p>Show me!</p> : <p>Else show me!</p>}
+```
+
+### Etiquetas dinámicas
+
+También puedes usar etiquetas dinámicas ajustando una variable en una etiqueta HTML o el import de un componente.
+
+```astro title="src/components/DynamicTags.astro" /Element|(?<!My)Component/
+---
+import MyComponent from "./MyComponent.astro";
+const Element = 'div'
+const Component = MyComponent;
+---
+<Element>Hello!</Element> <!-- renders as <div>Hello!</div> -->
+<Component /> <!-- renders as <MyComponent /> -->
+```
+
+Cuando se usan etiquetas dinámicas:
+
+- **Los nombres de las variables deben estar capitalizados** Por ejemplo, usa `Element`, y no `element`. Del contrario, Astro intentará renderizar el nombre de tu variable como una etiqueta HTML literal.
+
+- **Hidratación directivas no está soportada** Cuando se usa [`client:*` hydrataciones directivas](/es/core-concepts/framework-components/#hydrating-interactive-components), Astro necesita saber que componentes empaquetar para producción, y el patrón de la etiqueta dinámica previene esto de funcionar.
+
+### Fragmentos & Múltiples Elementos
```

Comment:
Tendrias que reemplazar esta seccion con esta nueva
```suggestion
### Fragments

Astro supports using either `<Fragment> </Fragment>` or the shorthand `<> </>`.

Fragments can be useful to avoid wrapper elements when adding [`set:*` directives](/en/reference/directives-reference/#sethtml), as in the following example:

```astro title="src/components/SetHtml.astro" "Fragment"
---
const htmlString = '<p>Raw HTML content</p>';
---
<Fragment set:html={htmlString} />
```
```